### ISSUE TITLE
Screen reader does not read matrix

### ISSUE THREAD

**Issue by RachelleBowen** [mathjax/MathJax#1875]
Our testing has revealed that matrices are not being read out by the screenreader. 

The screen reader finds a table and says 'left bracket' followed by information about the table - two columns, two rows, but no numbers are read out.

Browser: Firefox
Screenreader: JAWS for Windows

MathML:

![image](https://user-images.githubusercontent.com/34082635/33356177-d23785d8-d506-11e7-8087-510b236f6456.png)

**Comment by pkra** [comment]
Thanks for the report. It sounds like you are describing an issue with JAWS so you might want to file a bug at https://github.com/FreedomScientific/VFO-standards-support/issues rather than here.

If this is about MathJax rendering specifically, then please provide more detailed information, including a page to a live example and more information on the browser, OS and AT versions. Thanks.

**Comment by pkra** [comment]
On second thought, maybe this is just about how to use JAWS? JAWS hacks into IE and FF to override MathJax's rendering. JAWS expects users to press enter after arriving at an expression; this then opens up an external application for exploration. 

We generally recommend using our built-in voicing and exploration tool which can be reached via the MathJax menu => Accessibility => Explorer; cf. http://docs.mathjax.org/en/latest/misc/accessibility-features.html#mathjax-accessibility-extensions. This gives superior voicing and exploration without resorting to hackery and external applications. 

However, given how JAWS hacks around in the browser, this can get broken by JAWS.
